lib/vagrant-google/action/read_ssh_info.rb

```diff
@@ -42,11 +47,19 @@ def read_ssh_info(google, machine)
             return nil
           end
 
-          # Read SSH network info
-          return {
-            :host => server.public_ip_address,
-            :port => 22
-          }
+          if use_private_ip then
```

Comment:
Linter is complaining:
```
temikus λ rake lint
...
lib/vagrant-google/action/read_ssh_info.rb:56:11: C: Use the return of the conditional for variable assignment and comparison.
          if use_private_ip then
          ^^^^^^^^^^^^^^^^^^^^^^
lib/vagrant-google/action/read_ssh_info.rb:56:29: C: Do not use then for multi-line if.
          if use_private_ip then
                            ^^^^
```
The more important one is the first, see: http://www.rubydoc.info/github/bbatsov/RuboCop/RuboCop/Cop/Style/ConditionalAssignment

I would indeed suggest not setting the whole set of parameters via if condition, but rather change the address if private_ip is found, something like:
```
          ssh_info = {
              :host => server.public_ip_address,
              :port => 22
          }

          ssh_info[:host] = server.private_ip_address if use_private_ip
```